src/type/__tests__/enumType.js

```diff
@@ -59,7 +66,31 @@ describe('Type System: Enum Values', () => {
     }
   });
 
-  var schema = new GraphQLSchema({ query: QueryType });
+  var MutationType = new GraphQLObjectType({
+    name: 'Mutation',
+    fields: {
+      favoriteEnum: {
+        type: FavoriteType,
+        args: { color: { type: ColorType } },
+        resolve(value, color) { return color; }
```

Comment:
Here's the error I get when destructuring the `color` from the arguments object:

```
  1 failing

  1) Type System: Enum Values accepts enum literals as input arguments to mutations:

      AssertionError: expected { Object (data) } to deeply equal { Object (data) }
      + expected - actual

       {
         "data": {
           "favoriteEnum": {
      -      "color": [null]
      +      "color": "GREEN"
           }
         }
       }
      
      at Object.callee$1$0$ (src/type/__tests__/enumType.js:88:17)
```